docs/resources/core/private_ip.md

```diff
@@ -1,5 +1,11 @@
 # oci\_core\_private_ip
 
+[PrivateIp Reference][b236f15a]
+
+  [b236f15a]: https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/20160918/PrivateIp/ "PrivateIpReference"
+
+**dan: does this get IP info, or does it provide a private IP resource/**
```

Comment:
Provides a resource. PrivateIP is a first class object with its own CRUD operations.

Reply:
done